x-pack/plugins/uptime/public/components/monitor_management/monitor_list/monitor_list.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiBasicTable, EuiPanel, EuiSpacer } from '@elastic/eui';
+import { MonitorManagementList as MonitorManagementListState } from '../../../state/reducers/monitor_management';
+
+interface Props {
+  pageSize: number;
+  setPageSize: (val: number) => void;
+  monitorList: MonitorManagementListState;
+}
+
+export const MonitorManagementList: ({
+  monitorList: { list, error, loading },
+  pageSize,
+  setPageSize,
+}: Props) => any = ({ monitorList: { list, error, loading }, pageSize, setPageSize }) => {
+  const items = list.monitors ?? [];
+
+  const columns = [
```

Comment:
All of the types for the render functions should be replaced with types from the runtime monitor type